### ISSUE TITLE
Land initial OAuth Instance APIs in FxA

### ISSUE THREAD

**Issue by rfk** [mozilla/application-services#390]
This is a tracking issue for https://github.com/mozilla/fxa-auth-server/pull/2715 so that we can include it in our sprint planning on the a-s waffleboard.

We need to get the initial OAuth Instance Metadata APIs landed in FxA - maybe not the entire API, but enough to support basic device registration for reference-browser.

**Comment by vladikoff** [comment]
from mtg: blocked on server-side changes..

**Comment by rfk** [comment]
Closing this in favour of new approach using existing devices api.
